components/global/CtBetaAlert.vue

```diff
@@ -0,0 +1,10 @@
+<script>
+export default {}
+</script>
+
+<template>
+  <Alert type="warning">
+    <Icon name="exclamation-triangle"></Icon> Component Testing is currently in
+    beta.
```

Comment:
This is sufficient messaging. @cowboy did you think any other details are necessary?

Reply:
Seems fine, unless we want to explain _why_ anywhere.